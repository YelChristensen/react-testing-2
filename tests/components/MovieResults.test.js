import React from "react";
import renderer from "react-test-renderer";
import MovieResults from "../../src/components/MovieResults";



describe("MovieResults", () => {
  test("matches the snapshot", () => {
    const movieArr = [
      {
        imdbID: "aa",
        Year: 2017,
        Title: "We love testing",
        Poster: "xx"
      },
      {
        imdbID: "bb",
        Year: 2018,
        Title: "We love resting",
        Poster: "XXX"
      }
    ];
    const tree = renderer.create(<MovieResults movies={movieArr}/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
