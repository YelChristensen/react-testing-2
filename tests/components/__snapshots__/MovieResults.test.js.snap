// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieResults matches the snapshot 1`] = `
<div>
  <ul>
    <li>
      <h2>
        We love testing
         - 
        2017
      </h2>
      <div>
        <span>
          Movie has 
          0
           likes
        </span>
        <button
          onClick={[Function]}
        >
          Like
        </button>
      </div>
      <img
        src="xx"
      />
    </li>
    <li>
      <h2>
        We love resting
         - 
        2018
      </h2>
      <div>
        <span>
          Movie has 
          0
           likes
        </span>
        <button
          onClick={[Function]}
        >
          Like
        </button>
      </div>
      <img
        src="XXX"
      />
    </li>
  </ul>
</div>
`;
